tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"compilerOptions": {
 		"module": "commonjs",
 		"outDir": "lib",
-		"target": "ES6",
+		"target": "ES2018",
```

Comment:
It should be fine to use `es2020` for node 12

Reply:
According to [node.green](https://node.green/#ES2020-features-String-prototype-matchAll-throws-on-non-global-regex), 2019 is the latest standard node12 is at parity with so I've bumped to that.